hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-applicationhistoryservice/src/test/java/org/apache/hadoop/yarn/server/timeline/webapp/TestTimelineWebServices.java

```diff
@@ -81,11 +82,9 @@
   private static TimelineStore store;
   private static TimelineACLsManager timelineACLsManager;
   private static AdminACLsManager adminACLsManager;
-  private long beforeTime;
+  private static long beforeTime;
 
-  private Injector injector = Guice.createInjector(new ServletModule() {
-
-    @SuppressWarnings("unchecked")
```

Comment:
It looks to me that the javac warning is related because the patch removes `@SuppressWarnings("unchecked")`.